e2e/gitlab.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2017 HootSuite Media Inc.
```

Comment:
we should have a meta PR to cleanup license headers :)

Reply:
Yeah let's discuss what we want to do here and have a script that does the right thing